lib/components/button.vue

```diff
@@ -1,87 +1,96 @@
 <template>
-    <button :class="classList"
+    <button v-bind="conditionalLinkProps"
```

Comment:
I found this looking for proxying events dynamically. That eventually lead me to that `@event.native`, but this was an awesome find along the way!